cirq-google/cirq_google/study/device_parameter.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, Optional, Sequence
+import dataclasses
+from typing_extensions import Protocol
+import cirq
+
+
+class SupportsParameter(Protocol):
```

Comment:
Should this be `SupportsDeviceParameter`?

Reply:
Done.